### ISSUE TITLE
1.12.2 - errros

### ISSUE THREAD

**Issue by minecraft7net** [freddedotme/BuildSubmission#7]
i have this error on startup

https://pastebin.com/fhLpJVpi

**Comment by freddedotme** [comment]
I assume by your title that you run 1.12.2, yeah? I haven't maintained this code in 2 years but I'll have a look once I have some free time.

**Comment by minecraft7net** [comment]
yes i have creative server on 1.12.2 if it possible please also update to 1.13.2
